### PR TITLE
CPDP 1616 Add period of admin to list of approved for representation …

### PR DESCRIPTION
**Pull request by sam-miele**
…answers

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala [hunk 1]
```diff
@@ -368,12 +368,12 @@ object CompleteReturn {
   ): Option[Option[CompleteRepresenteeAnswers]] =
     (individualUserType, representeeAnswers) match {
       case (
-            Some(Capacitor) | Some(PersonalRepresentative),
+            Some(Capacitor) | Some(PersonalRepresentative) | Some(PersonalRepresentativeInPeriodOfAdmin),
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala:371]: you could replace all of these with just `Some(RepresentativeType)` to make it shorter

Comment (sam-miele) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala:371]: From the looks of it we want to exclude `Self`

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala:371]: `RepresentativeType` does exclude `Self` - it's basically everything other than `Self`

Comment (sam-miele) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/returns/CompleteReturn.scala:371]: Oh sorry! I misread the file, that extends `IndividualUserType`. Updating now.

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**